src/Xamarin.Android.Build.Tasks/Utilities/MonoAndroidHelper.cs

```diff
@@ -544,7 +548,7 @@ internal static IEnumerable<ITaskItem> GetFrameworkAssembliesToTreatAsUserAssemb
 			"Mono.Posix.dll",
 		};
 		// MUST BE SORTED CASE-INSENSITIVE
```

Comment:
we can probably remove this comment now, since we're no longer doing a binary search...